icu4c/source/i18n/measunit_extra.cpp

```diff
@@ -777,6 +775,7 @@ MeasureUnit SingleUnitImpl::build(UErrorCode& status) const {
     return std::move(temp).build(status);
 }
 
+const char *SingleUnitImpl::getIdentifier() const { return gSimpleUnits[index]; }
```

Comment:
Multi-line: check/change clang-format options...

Reply:
Reformatted.
The .clang_format option for this is:

    AllowShortFunctionsOnASingleLine: false